config/routes.rb

```diff
@@ -9,6 +9,7 @@
   namespace :api do
     namespace :v1 do
       resources :podcasts, only: [:index, :create, :show]
+      resources :reviews, only: [:index]
```

Comment:
do we need this path? I thought since we're not showing reviews at a unique path, only within the podcast show page, we don't need a dedicated route?

Reply:
if it is needed, shouldn't it be nested underneath podcasts, within a do block?